src/antsibull/data/ansible-readme.rst

```diff
@@ -14,6 +14,19 @@ deployment, cloud provisioning, ad-hoc task execution, network automation, and m
 orchestration. Ansible makes complex changes like zero-downtime rolling updates with load balancers
 easy. More information on the Ansible `website <https://ansible.com/>`_.
 
+This is the ``ansible`` community package.
+``ansible`` contains a set of independent Ansible collections that are currated
+by the community, and it pulls in
+`ansible-core <pypi.org/project/ansible-core/>`_.
+The ``ansible-core`` package contains the core engine and CLI tools (such as
+``ansible``, ``ansible-playbook``) while ``ansible`` contains extra modules,
```

Comment:
I find the differing use of ``ansible`` back to back a little confusing without qualifying terms surrounding it, I'd like to suggest something similar to the following as a change:

```markdown
The ``ansible-core` python package contains the core runtime and CLI tools (such as
``ansible``, ``ansible-playbook``) while the ``ansible`` python package contains extra modules,
plugins, and roles.
```

Reply:
https://github.com/ansible-community/antsibull/pull/474/commits/b9713d0af649f318e7e5e77e1fcab96f4cf1e30b